ZupIT/beagle-web-core
# Fix: Unnecessary render

**- What I did**
Remove the possibility to do unnecessary render using the `doTemplateRender` method.

**- How I did it**
Fixing how the core sets the id of a cloned Tree component.

**- How to verify it**
Using lists, especially with infinite scroll, check if the ids of each element are changing when you set a new chunk of source to the main data source.

**- Description for the changelog**
